src/AccessibilityInsights.SharedUx/Controls/TestTabs/ColorContrast.xaml.cs

```diff
@@ -119,13 +120,29 @@ private void RunAutoCCA(Bitmap bitmap)
 
             if (pair == null)
             {
+                Logger.PublishTelemetryEvent(
+                    TelemetryAction.ColorContrast_AutoDetect,
+                    new Dictionary<TelemetryProperty, string>
+                    {
+                        [TelemetryProperty.BitmapSize] = (bitmap.Width * bitmap.Height).ToString(CultureInfo.InvariantCulture),
+                        [TelemetryProperty.Confidence] = "None"
```

Comment:
Is there an `Enum` for confidence level strings?

Reply:
Yes. https://github.com/microsoft/axe-windows/blob/main/src/Desktop/ColorContrastAnalyzer/IColorContrastResult.cs#L7